bowling/tests/lane.ut.cpp

```diff
@@ -2,7 +2,40 @@
 
 #include "lane.hpp"
 
-TEST_CASE()
+#include <string>
+
+SCENARIO("Lane class constructor takes one string type argument", "[creating lane object]")
 {
-    REQUIRE(1 == 1);
+    GIVEN("Argument as a name of file")
+    {
+        std::string n = "lane";
+        std::string n1 = "lane1.txt";
+        std::string n2 = "lane2.doc";
+        std::string n3 = "lane3.odt";
+        std::string n4 = "lane4.cpp";
+        std::string n5 = "";
+        std::string n6 = ".txt";
```

Comment:
Nice to have: Instead of creating bunch of names, you could use GENERATE property of catch2.
You could generate pair of input name and lane name, e.g.:

`auto [testFileName, expectedIsLaneName] =
        GENERATE(std::make_pair("lane1.txt", "lane1"),
                 std::make_pair("lane3.odt", "lane3"))`

and then use these values in test